README.md

```diff
@@ -97,6 +97,7 @@ PIWIK_URL | Matomo URL | `http://url`
 PIWIK_SITE_ID | Matomo Site ID | 1
 PIWIK_START_GOAL_ID | Matomo Start goal ID | 2
 PIWIK_UPDATE_START_GOAL_ID | Matomo Update Start goal ID | 3
+PIWIK_REMOVE_START_GOAL_ID | Matomo Remove Start goal ID | 4
```

Comment:
Should match ids to Docker values, we think.

Reply:
updated to the docker values